Source/UI/Packages/Installing/View Controllers/ProgressController.mm

```diff
@@ -64,6 +64,18 @@ - (void) viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
 }
 
+- (void) justClose
+{
+    UpdateExternalStatus(0);
+    [delegate_ returnToCydia];
+    //[[[self navigationController] parentOrPresentingViewController] dismissModalViewControllerAnimated:YES];
+}
+
+- (void) applyRightButton
```

Comment:
Could we move this method right next to -leftButton?
Also, it's not working. I tapped it, -justClose got called, and the controller wasn't dismissed.